components/WhyChooseUs/index.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from "next/link";
+import React from "react";
+import CardContainer from "./CardContainer";
+
+const WhyChooseUs = () => {
+  return (
+    <section className="px-8 py-12 relative bg-backgroundColor  max-w-7xl mx-auto">
+      <h2 className="text-h2">Why Choose Us</h2>
+      <p className="font-light">
+        The products we provide are different from others
+      </p>
+      <div className="flex flex-col justify-center items-center">
+        <CardContainer />
+        <Link
+          href="/design"
```

Comment:
Same here. The Link here should guide the user to [www.design.courtcanva.com](http://www.design.courtcanva.com/) instead of [www.courtcanva.com/design](http://www.courtcanva.com/design) in my view. The href="/design" here refers to [www.courtcanva.com/design](http://www.courtcanva.com/design). It shoud be fixed if my logic is correct.